mapserver/django/django_project/mapserver/urls.py

```diff
@@ -0,0 +1,13 @@
+# coding=utf-8
+
+from django.conf.urls import url
+from rest_framework.urlpatterns import format_suffix_patterns
+
+from mapserver.api.wms import wms, ows
+
+urlpatterns = [
+    url(r'^wms', wms, name='wms'),
```

Comment:
@meomancer Previously `^wms/` and the redirect from `wms?` to `wms/?` is not correct. So I revert it back.
Please use url scheme from our new nginx where our django backend is located in `http://fbf.test/fba/wms` it is under `/fba` if accessed from the frontend. 
Make sure to update your configuration so the changes applies.